test/wasm2js/target_js.wast

```diff
@@ -0,0 +1,29 @@
+ (func $0 (param $0 i32) (result i32)
+  (unreachable)
+ )
+ (func $1 (param $0 i32) (param $1 i32)
+  (i32.store
+   (i32.const 1)
+   (i32.xor
```

Comment:
I think we only need the xor for this test? We could have one function that returns an i32, which is the xor.

Reply:
As noted above, if I remove the table and simplify the function to xor, is there another way to retain the function at all? Anything further than the revised wast noted in my other reply seems to excise the function entirely at wasm2js -O1.

Reply:
Ah, yes, you need to export the function. Without that, only the table was keeping it alive...

Here is a minified version you can use:
```wat
(module
  (func $func (export "func") (param $x i32) (result i32)
    ;; Do not turn xor into something that does not express well in JS
    ;; when optimizing.
    (i32.xor
      (i32.shl
        (i32.const 1)
        (local.get $x)
      )
      (i32.const -1)
    )
  )
)
```

Reply:
Many thanks. Test case modified accordingly.